<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Console Regression Test Automation - UAT Test Suite" duration-ms="161469" started-at="2018-09-13T04:56:53Z" finished-at="2018-09-13T04:59:34Z">
    <groups>
    </groups>
    <test name="Regression Test Cases" duration-ms="161469" started-at="2018-09-13T04:56:53Z" finished-at="2018-09-13T04:59:34Z">
      <class name="com.consoleregression.testcases.SalesDBJourneyTest">
        <test-method status="FAIL" signature="verify_ComDomain_and_Office365_Order_InSalesDB(java.lang.String)[pri:3, instance:com.consoleregression.testcases.SalesDBJourneyTest@146ba0ac]" name="verify_ComDomain_and_Office365_Order_InSalesDB" duration-ms="161439" started-at="2018-09-13T04:56:53Z" finished-at="2018-09-13T04:59:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[uat1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[productsetup2 status:  expected [approved] but found [setup hosting package failed]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: productsetup2 status:  expected [approved] but found [setup hosting package failed]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.consoleregression.testcases.SalesDBJourneyTest.verify_ComDomain_and_Office365_Order_InSalesDB(SalesDBJourneyTest.java:291)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_ComDomain_and_Office365_Order_InSalesDB -->
      </class> <!-- com.consoleregression.testcases.SalesDBJourneyTest -->
    </test> <!-- Regression Test Cases -->
  </suite> <!-- Console Regression Test Automation - UAT Test Suite -->
</testng-results>
